### PR TITLE
Lazy loading

### PR DESCRIPTION
**Pull request by kainanaina**
1) Adds `withPreload` HOC.
2) Added webpackBundleAnalyzer for build, to check detailed bundle size information (generates webpack-report.html in root directory on every build run).
3) Added code for dynamic code splitting for routing. Currently disabled (commented AsyncChunks file in utils, doesn't impacts bundle), because our pages barely have any content. Will test it again once we'll add some good amount of code.
4) Some small webpack config tweaks.

### CODE REVIEW

File: src/Utils/withPreload.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { withApollo } from '@apollo/react-hoc';
+import { getQuery } from '$queries/mapper';
+
+// preloads static queries (which require no params) into cache after certain delay
+export default (Component, queryNames = [], delay = 5000) => (
+  withApollo(({ client, ...props }) => {
```
Comment (kainanaina) [inline src/Utils/withPreload.js]: `withApollo` hoc returns `client` which allows you to directly run queries (which you can't easily do with hooks and HOCs).

Comment (SaiCharanMahadevan) [reply src/Utils/withPreload.js]: hmmm interesting, lets talk about this tomorrow. Have to read about the original problem about this `run queries (which you can't easily do with hooks and HOCs).`

File: src/Pages/Home/Component.js [hunk 2]
```diff
@@ -1,47 +1,53 @@
...
-            ? null // replace with loading placeholders
-            : (
-              <ContentSections
-                order={homeContent.order}
-                data={homeContent.data}
-              />
-            )
-          }
+export default ({ homeContent, history }) => {
+  const [searchQuery, setSearchQuery] = React.useState('');
```
Comment (SaiCharanMahadevan) [inline src/Pages/Home/Component.js]: hahahaha you using hooks nice

File: src/Utils/withPreload.js (hunk 1 above)
Comment (kainanaina) [reply src/Utils/withPreload.js]: It's not a problem, it's just how it works. You can run hooks outside of react components and HOC's work as wrappers which suppose to send you new props, which we don't need.

**Pull request merged**